schema/types/suricata.schema

```diff
@@ -97,17 +97,39 @@ type suricata.dcerpc = suricata.component.common + record {
   }
 }
 
+// https://github.com/OISF/suricata/blob/master/rust/src/dhcp/logger.rs
 type suricata.dhcp = suricata.component.common + record {
   dhcp: record {
-    type: string,
+    type: string, // consider switching to enum { request, reply, <unknown> }
```

Comment:
Please do so if we're changing types of existing fields anyways.

Reply:
We can't because "<unknown>" is not a valid token in an enum alternatives list.